zerver/tests/test_create_video_call.py

```diff
@@ -36,187 +36,193 @@ def setUp(self) -> None:
             }
         )
 
-    def test_register_video_request_no_settings(self) -> None:
+    def test_register_zoom_user_auth_request_no_settings(self) -> None:
         with self.settings(VIDEO_ZOOM_CLIENT_ID=None):
             response = self.client_get("/calls/zoom/register")
             self.assert_json_error(
                 response,
                 "Zoom credentials have not been configured",
             )
 
-    def test_register_video_request(self) -> None:
+    def test_register_zoom_user_auth_request(self) -> None:
         response = self.client_get("/calls/zoom/register")
         self.assertEqual(response.status_code, 302)
 
     @responses.activate
-    def test_create_zoom_video_and_audio_links(self) -> None:
-        responses.add(
-            responses.POST,
-            "https://zoom.us/oauth/token",
-            json={"access_token": "oldtoken", "expires_in": -60},
-        )
+    def test_create_zoom_user_auth_video_and_audio_links(self) -> None:
+        with self.settings(VIDEO_ZOOM_ACCOUNT_ID=None):
+            responses.add(
+                responses.POST,
+                "https://zoom.us/oauth/token",
+                json={"access_token": "oldtoken", "expires_in": -60},
+            )
```

Comment:
For tests, one may want to split `TestVideoCall` into having a separate `TestZoomIntegration` class, and have a subclass that just changing the `VIDEO_ZOOM_ACCOUNT_ID` in order to run the same tests with each integration. May or may not need an abstract class, depending whether one wants any tests just for one code path or the other.

Reply:
I did split out the Big Blue Button and Zoom tests to have a separate subclass for testing in a prep commit.

And then when adding the Zoom Server to Server integration, added the `override_settings` decorator from Django for the Zoom User Auth integration tests.